implementations/code/code_changes_git.py

```diff
@@ -0,0 +1,78 @@
+from datetime import datetime
+from itertools import groupby
+import conditions
+import utils
+from commit_git import CommitGit
```

Comment:
Please, leave a blank line. See [coding standard on import](https://www.python.org/dev/peps/pep-0008/#imports)

Reply:
Oops! Overlooked that. Made the change.